pyartcd/pyartcd/locks.py

```diff
@@ -195,10 +205,5 @@ async def run_with_lock(coro: coroutine, lock: Lock, lock_name: str, lock_id: st
 
         async with await lock_manager.lock(resource=lock_name, lock_identifier=lock_id):
             return await coro
-
-    except LockError as e:
-        lock_manager.logger.error('Failed acquiring lock %s: %s', lock_name, e)
```

Comment:
This is being logged when unlock() raises an exception, so `Failed acquiring lock` isn't accurate. Move logging error to the lock method.